Analytics/Server/TryAppService.Data/Models/UserPuid.cs

```diff
@@ -36,4 +36,7 @@ public override bool Equals(object obj)
                 .Equals(this.Puid, userPuid.Puid);
         }
     }
+    public class UserPuid2 : UserPuid
```

Comment:
I have a separate DB 'tryitnowanalytics2' which has indexes on it . It references tables  in the 'tryitnowanalytics'  DB directly and populates `tablename2` with the right datatype. Was adding them for the analytics site to pick up. Will remove them. Plus this wont be needed after adding mixpanel